active_fulfillment.gemspec

```diff
@@ -22,7 +22,12 @@ Gem::Specification.new do |s|
   s.add_dependency('activesupport', '>= 3.2.9', '< 5.1.0')
   s.add_dependency('builder', '>= 2.0.0')
   s.add_dependency('active_utils', '~> 3.0')
-  s.add_dependency('nokogiri', '~> 1.6')
+
+  if RUBY_VERSION < '2.1'
```

Comment:
You can't branch ruby version inside the `.gemspec` file. That would mean a different gem would be build based on the ruby building it.

If we have to bump a dependency version that's fine. Let's change the minor version of the gem and update the dependency to be the specific version we need.